src/main/resources/messages.properties

```diff
@@ -126,6 +126,9 @@ public.login.error.locked.title=There is a problem with your account login detai
 public.login.error.locked.instruction=Your account is locked due to too many unsuccessful attempts.<br>You can <a href="{0}">reset your password</a> to unlock your account.
 public.login.error.suspended.title=There is a problem with your account login details
 public.login.error.suspended.instruction=Your account has been blocked. Contact us to get help signing in.
+#TODO update once we get the proper message back from hmcts ux
+public.login.error.retired.title=There is a problem with your account login details
+public.login.error.retired.instruction=Your account has been deactivated. You have been sent a registration email.
```

Comment:
I know it's a placeholder message, but I think it should say something more like:
"Your account has been deactivated due to inactivity. Please reset your password using the link sent to your email to reactivate it."

Reply:
@dfourn applied suggested changes